INSTALL.rst

```diff
@@ -95,7 +95,18 @@ code).
 which modifies how the wasm and glue code are built and how they behave. An ``-s``
 should precede each flag.
 
-For example, to generate modularized glue code, use:
+For example, to generate a HTML page, use:
+
+.. code:: bash
+
+  ./configure.sh --static --static-binary --auto-download --wasm=HTML --name=prod
+
+  cd prod
+  make            # use -jN for parallel build with N threads
+
+After that, you can run ``python -m http.server`` within ``prod/bin``, open http://0.0.0.0:8000/cvc5.html with Chrome to visualize the page generated by Emscripten, write down a valid SMTLIB input, and press ESC twice to obtain its output.
```

Comment:
Is this only working with Chrome or is it supposed to be working with any browser?